gamemodes/terrortown/gamemode/client/cl_help_data.lua

```diff
@@ -0,0 +1,253 @@
+---
+-- @author Mineotopia
+
+HELP_DATA = {}
+
+local HELP_DATA_OBJECT = {}
+
+---
+-- Binds data table to the @{HELP_DATA} object
+-- @param table data The data table with all submenues
+-- @return @{HELP_DATA} The object to be used in the hook
+-- @internal
+-- @realm client
+function HELP_DATA:BindData(menuTbl)
+	self.menuTbl = menuTbl or {}
+
+	return self
+end
+
+---
+-- Creates a new submenu in the main menu
+-- @param string id The unique ID of the submenu
```

Comment:
ID or id, please be consistent

Reply:
I mean, the variable is called `id` but in text you write `ID` since it is an abbreviation

Reply:
No that's fine, but you used `id` and `ID` in the documentation :)

Reply:
oh, okay